pkg/render/compliance.go

```diff
@@ -463,18 +462,23 @@ func (c *complianceComponent) complianceServerClusterRole() *rbacv1.ClusterRole
 				Resources: []string{"globalreporttypes", "globalreports"},
 				Verbs:     []string{"get", "list", "watch"},
 			},
-			{
-				APIGroups: []string{"authentication.k8s.io"},
-				Resources: []string{"tokenreviews"},
-				Verbs:     []string{"create"},
-			},
 			{
 				APIGroups: []string{"authorization.k8s.io"},
 				Resources: []string{"subjectaccessreviews"},
 				Verbs:     []string{"create"},
 			},
 		},
 	}
+
+	if c.installation.Spec.ClusterManagementType == operatorv1.ClusterManagementTypeManagement {
```

Comment:
Will this also work for standalone clusters?

Reply:
Yes, it should. Let me give an example:

Standalone
- Jane uses UI and goes to compliance page
- Request is made with her header
- Jane has a role that includes authenticationReviews/create
- Compliance server sends this header to api-server in the form of an authenticationReviews/create call

Now Management: Jane wants to access cluster X
- Same 4 steps
- Now compliance server needs data from other cluster
- A request on behalf of the compliance-server is made to voltron, to be tunneled to X
- Voltron will then use the header that belongs to the compliance-server and make the authenticationReviews/create call

Reply:
You might wonder why es-proxy does not need this permission then. (tigera-manager SA)
I did too.
It turns out that the rbac checks done in the other cluster are on behalf of Jane again. A special config is made for Jane in the rbac code, and so it is Jane who will be impersonated.

Reply:
Makes sense. Thanks for explaining the reason.